alpha/apps/kaltura/lib/storage/kFileSyncUtils.class.php

```diff
@@ -1163,6 +1168,16 @@ public static function createSyncFileForKey ( $rootPath, $filePath, FileSyncKey
 		return $currentDCFileSync;
 	}
 
+	public static function setActualFileSizeOnAsset($currentDCFileSync)
+	{
+		$asset = assetPeer::retrieveById($currentDCFileSync->getObjectId());
+		if ($asset && $asset->getType() == assetType::FLAVOR && $currentDCFileSync->getFileSize())
+		{
+			$asset->setActualFileSizeOnDisk((int) ($currentDCFileSync->getFileSize() / 1024));
```

Comment:
Why do we need the size in kbytes, already have it  and its not accurate.
Lets leave the size as is, in bytes